modules/dns_libvirt/dns_libvirt_main.rb

```diff
@@ -0,0 +1,60 @@
+require 'dns_libvirt/libvirt_dns_network'
+require 'rexml/document'
+require 'dns_common/dns_common'
+
+module Proxy::Dns::Libvirt
+  class Record < ::Proxy::Dns::Record
+    include Proxy::Log
+    include Proxy::Util
+
+    attr_reader :libvirt_network, :network
+
+    def initialize(options = {})
+      @network = options[:network] || Proxy::Dns::Libvirt::Plugin.settings.network
+      @libvirt_network = options[:libvirt_network] || ::Proxy::LibvirtNetwork.new(
```

Comment:
Should be ::Proxy::LibvirtDNSNetwork. Same comment about namespace as for dhcp_libvirt_main.rb.